commit 7fbde4f5cbf4d989ccd477e63a83d84ceccb2439
Author: Todd Rinaldo <toddr@cpan.org>
Date:   Thu Feb 4 12:24:31 2010 -0600

    Remove use of v and qv notation which breaks install of older perls

diff --git a/Build.PL b/Build.PL
index 15bb927..290c4ce 100644
--- a/Build.PL
+++ b/Build.PL
@@ -48,7 +48,7 @@ my $build = $class->new(
     license         => 'bsd',
     requires        => {
         # Core requirements:
-        perl            => 'v5.6',
+        perl            => '5.006',
         version         => 0,
         Error           => 0,
         NetAddr::IP     => '4',
@@ -63,7 +63,7 @@ my $build = $class->new(
         Module::Build   => '0.2805',
         Test::More      => 0,
         Net::DNS::Resolver::Programmable
-                        => 'v0.002.1',
+                        => '0.002001',
     },
     script_files    => [
         'bin/spfquery'
diff --git a/CHANGES b/CHANGES
index b7c4303..838d24a 100644
--- a/CHANGES
+++ b/CHANGES
@@ -4,6 +4,11 @@
 #   ! = Changed something significant, or removed a feature
 #   * = Fixed a bug, or made a minor improvement
 
+--- 2.008 (2010-02-04 12:00)
+
+  Miscellaneous:
+  * Fix version numbers to provide perl compaibility prior to 5.8
+
 --- 2.007 (2009-10-31 21:00)
 
   Mail::SPF:
diff --git a/lib/Mail/SPF.pm b/lib/Mail/SPF.pm
index 67d1e5c..4a62632 100644
--- a/lib/Mail/SPF.pm
+++ b/lib/Mail/SPF.pm
@@ -17,11 +17,11 @@ Mail::SPF - An object-oriented implementation of Sender Policy Framework
 
 =head1 VERSION
 
-2.007
+2.008
 
 =cut
 
-use version; our $VERSION = qv('2.007');
+use version; our $VERSION = '2.008';
 
 use warnings;
 use strict;
